torchgen/model.py

```diff
@@ -1185,16 +1421,121 @@ def strip_ret_annotation(r: Return) -> Return:
                 ),
                 overload_name="",  # stripped
             ),
-            arguments=self.arguments.signature(strip_default=strip_default),
-            returns=tuple(map(strip_ret_annotation, self.returns)),
+            arguments=args_sig,
+            returns=returns,
         )
 
     def view_signature(self) -> "FunctionSchema":
         return self.signature(strip_view_copy_name=True)
 
+    def self_to_out_signature(self) -> "FunctionSchema":
+        # Generating an out= schema from an inplace schema.
```

Comment:
it's weird that you have to do this, in my ideal world there wouldn't be ops which we only have inplace and not others. But alas...